preprocess/prepro_utils.py

```diff
@@ -3,6 +3,14 @@
 import logging
 import unicodedata
 from nltk.stem.porter import PorterStemmer
+import string
+import spacy
+from nltk.stem import SnowballStemmer
+
+# Load spacy model for stemming
+stemmer_spanish = SnowballStemmer("spanish")
```

Comment:
why load the spanish news dataset here and the Spanish stemmer?

Reply:
Yes, I think it's an upload error.

This becomes a bug because line 293 (prepro_utils.py) shows that even the English dataset uses the Spanish stemmer.

I have solved this problem. Thanks Lukas!